lsst-epo/investigations-client
# Validated number questions in multipart type

## What this change does ##

Add validated numbers to multipart questions and do some cleanup and extraction of multipart questions

## Notes for reviewers ##

Client + API changes
## Testing ##

- Create a multipart question in Craft, add a number with validations
- Observe on the page that validations work as intended (additionally check a standalone number question to see that it also still works)
- Observe that the question also displays in the review page and in XLSX

Resolves #171
